### PR TITLE
Modified stained glass description to reflect the fact it is a wall with a high window, changed all external walls to stone walls for cathedral (#49440)

### LINKED ISSUE #49440: Lower str of high stained glass window, so it could be smashed
# Describe the bug

`t_window_stained_green`/`t_window_stained_red`/`t_window_stained_blue` has an insane "str_min": 100, which is the same value of concrete terrain.
Even a .50 rifle can't break it, holy glasses!

# Steps To Reproduce

1. Find a Cathedral
2. Smash the ultra-holy-reinforced glass as a zombie did
3. Damn my hand hurts

# Expected behavior

Use the same value with Double glazed glass window(`t_double_pane_glass`) and most of glasses window.
```
"bash": {
      "str_min": 3,
      "str_max": 6,
...
```

More, add `glass_tinted` as its ingredient.
```
...
      "items": [ { "item": "glass_shard", "count": [ 34, 50 ] }, "item": "glass_tinted", "count": [ 8, 17 ] } ]
```

A broken stained glass symbolizes post-cataclysm era, smash it.

# Screenshots

圖片

# Versions and configuration

 - OS Version: 10.0 2009
 - Game Version: 17c44e4
 - Graphics Version: Tiles
 - Ingame language: English
 - Mods loaded: [
"dda",
"no_npc_food",
"cbm_slots",
"sees_player_retro"
]

# Additional context

### PR DESCRIPTION
**Pull request by ghost**
#### Summary
Bugfixes "Modified stained glass description to reflect the fact it is a wall with a high window, changed all external walls to stone walls for cathedral"

#### Purpose of change
Fixes #49440 by making clear original purpose of stained glass walls

#### Describe the solution
- Changed description of stained glass to make clear it is not a full window but a stone wall containing stained glass
- Changed bash strength to 80\300 in line with stone walls
- Added rocks into bash items same as stone walls
- Changed cathedral mapgen to stone walls instead of packed earth to standardize walls used for cathedral

#### Describe alternatives you've considered
Lower bash strength of stained glass to str_min 5 and str_max 8 as suggested by @I-am-Erk in  #49440 (original solution but modified after feedback)

#### Testing
Loaded into base game

#### Additional context

Cathedral

### CODE REVIEW

**Comment by Fris0uman** [comment]
The value were hight because it was assume that you're actually smashing stone and that the windows is way above ground and not reachable by smash.

**Comment by Broken27** [comment]
> The value were hight because it was assume that you're actually smashing stone and that the windows is way above ground and not reachable by smash.

If that's the intent the windows should be one z level above ground.

**Comment by chrispikula** [comment]
I think they meant a small stained glass window near the ceiling?  You find such in some churches.  
A full-tile stained glass window would be something else, which should be as easy to break as any other double or triple paned window.
(Oftimes modern stained glass windows have a solid sheet of normal glass on either side to make them more structurally sound and/or easier to clean.)

**Comment by occulti** [comment]
I see the logic here, I suppose the difference is between this one where the windows would clearly be represented one Z level up:

st-anthony-of-padua-church

And this one where the windows are reachable but there is a significant portion of wall beneath them:

Dx7S5ciUYAE7ZnS

I'm guessing from the comments it is the latter as if they are smashed you are able to climb through them?

I am happy to close this pull request if I have missed the original intention in an attempt to fix perceived bugs :)

**Comment by occulti** [comment]
Just a further thought on this, might be worth changing the name to "stone wall with high stained glass window" and adding rubble to the bash as well as glass to represent the whole wall coming down on bash?

**Comment by int-ua** [comment]
https://github.com/CleverRaven/Cataclysm-DDA/commit/0df1e05ef8d5c1d2a8a9fbd718c18be7177a4381 @Rivet-the-Zombie

**Review by Rivet-the-Zombie — APPROVED** [review]
This all looks good to me.

**Comment by actual-nh** [comment]
@Maleclypse, could you kick the tests to run? Thanks!

**Comment by Maleclypse** [comment]
Interestingly a test has to at minimum fail before I can kick it to make it rerun.  With only the LGTM tests running there is nothing for me to kick in my usual spot.

File: data/json/furniture_and_terrain/terrain-windows.json [hunk 1]
```diff
@@ -743,19 +743,19 @@
...
-      "items": [ { "item": "glass_shard", "count": [ 42, 67 ] } ]
+      "items": [ { "item": "glass_shard", "count": [ 42, 67 ] }, { "item": "rock", "count": [ 8, 18 ] } ]
     }
   },
   {
     "type": "terrain",
     "id": "t_window_stained_red",
-    "name": "high stained glass window",
-    "description": "Breathtaking craftsmanship of stained glass featuring an otherworldly radiant ruby flower blooming.",
+    "name": "stone wall woth high stained glass window",
```
Comment (anothersimulacrum) [inline data/json/furniture_and_terrain/terrain-windows.json:757]:
```suggestion
    "name": "stone wall with high stained glass window",
```

**Review by anothersimulacrum — APPROVED** [review]

**Comment by actual-nh** [comment]
Should also fix #50364.

**Comment by actual-nh** [comment]
Asking Kevin to take a look and hopefully do something about the stuck tests. This isn't the only PR it's happening on.

**Pull request merged**